bigquery/cloud-client/snippets.py

```diff
@@ -157,19 +125,10 @@ def list_rows(dataset_name, table_name, project=None):
     # Reload the table so that the schema is available.
     table.reload()
 
-    rows = []
-    page_token = None
-
-    # Load at most 25 results. You can change this to `while True` and change
-    # the max_results argument to load more rows from BigQuery, but note
-    # that this can take some time. It's preferred to use a query.
-    while len(rows) < 25:
-        results, total_rows, page_token = table.fetch_data(
-            max_results=25, page_token=page_token)
-        rows.extend(results)
-
-        if not page_token:
-            break
+    # Load at most 25 results. You can change the max_results argument to load
+    # more rows from BigQuery, but note that this can take some time. It's
+    # preferred to use a query.
+    rows = table.fetch_data(max_results=25)
```

Comment:
You may prefer to just consume the iterator right here:

```python
rows = list(table.fetch_data(max_results=25))
```

There is too much suspense between this and the `for row in rows` (when the request(s) actually get made)